docs/content/get-started/upload-launch.md

```diff
@@ -56,14 +56,13 @@ When launching a cloud deployment you must provide three parameters:
 * **the assembly name**, which identifies the worker assemblies to use. These are the assemblies you uploaded earlier so the name should match the one you passed to `spatial cloud upload`.
 * **a launch configuration**, which is a JSON file that declares the world and load balancing configuration.
 * **a name for your deployment**, which is used to label the deployment in the SpatialOS web Console. It consists of lower-case letters, digits, and `_` and can be up to 32 characters long.
+* **(optional) a region to deploy to**, which default to clusters located in the US. If you’re in Europe, add the `--cluster_region=eu` flag for lower latency.
```

Comment:
```suggestion
* **(optional) a region to deploy to**, which defaults to server clusters located in the USA. If you’re in Europe, add the `--cluster_region=eu` flag so the servers are in Europe. This gives you lower latency.
```

Reply:
This doesn't say what the "clusters" or that by using this flag you are deploying to servers in Europe.